tools/sloop.c

```diff
@@ -0,0 +1,84 @@
+/**
+ * Library of functions pertaining to Sloop shadows. Slooping is a way
+ * to simulate the built-in snooping functionality of EotL.
+ *
+ * @author gon
+ * 2017-05
+ */
+
+inherit AnsiTellCode;
```

Comment:
I don't think you're using AnsiTellCode anywhere in sloop.c. You want to manage your dependencies when they're used, even if that means you inherit AnsiTellCode in both sphere.c and isolationchamber.c.